member-service/src/main/java/com/hedvig/memberservice/commands/SuccessfulAuthenticationCommand.kt

```diff
@@ -0,0 +1,51 @@
+package com.hedvig.memberservice.commands
+
+import com.hedvig.external.bankID.bankIdTypes.CompletionData
+import com.hedvig.external.zignSec.client.dto.ZignSecIdentity
+import com.neovisionaries.i18n.CountryCode
+import org.axonframework.commandhandling.TargetAggregateIdentifier
+
+data class SuccessfulAuthenticationCommand(
+    @TargetAggregateIdentifier val id: Long,
+    val method: AuthMethod
+) {
+
+    val identity: Identity = when (method) {
```

Comment:
I'm not 100% what the difference is here really but this makes sense to me
```suggestion
    val identity: Identity
        get() = when (method) {
```